src/core/constants.js

```diff
@@ -436,6 +436,12 @@ export const UP_ARROW = 38;
  * @default source-over
  */
 export const BLEND = 'source-over';
+/**
+ * @property {String} REMOVE
+ * @final
+ * @default destination-out
+ */
+export const REMOVE = 'destination-out';
```

Comment:
one small thing. if we're adding a blendMode option here, can you also add it to this list in the blendMode() reference: https://github.com/processing/p5.js/blob/0.9.0/src/core/rendering.js#L236

Reply:
Good point thank you!